nixos/modules/misc/ids.nix

```diff
@@ -510,7 +510,6 @@ in
       #seeks = 148; # removed 2020-06-21
       prosody = 149;
       i2pd = 150;
-      systemd-coredump = 151;
```

Comment:
This should have stayed as a comment

Reply:
This was a straight revert of the listed commits -- it was never a comment.

Reply:
It would have been better if it wasn't a straight revert but turned into a comment explaining that 151 was shortly used for systemd-coredump that not someone else takes in the future.